cs3api4lab/api/cs3_share_api.py

```diff
@@ -230,7 +230,7 @@ def _shares_received_filter_by_resource(self, path):
         try:
             file_stat = self.file_api.stat_info(path)
         except FileNotFoundError:
-            return sharing.ListReceivedSharesRequest()
+            return sharing.ListSharesRequest()
```

Comment:
This is not really related to this PR, right?

Reply:
It's not, and it's actually incorrect, it should be ListReceivedSharesRequest